### PR TITLE
fix with deploy

### PR DESCRIPTION
**Pull request by agnbart**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
Based on the provided task requirements and code changes, it seems that the task requirements are not met. The requirement was to deploy the service and add the deployed app URL to the `readme.md` file. However, none of the provided code changes include modifications to the `readme.md` file.

Here's a brief review of the provided code changes:

1. **index.ts**: The changes here are related to the server setup. The CORS origin has been updated to use the server's port, and the server is set to listen on all interfaces instead of just 'localhost'. These changes are typically done when preparing for deployment, but they do not fulfill the task requirement of adding the deployed URL to the `readme.md`.

2. **package.json**: New dependencies have been added, likely for testing purposes. This is not related to the task requirements.

3. **tests/card-*.test.ts**: Imports for `before` and `after` from "node:test" have been added in all test files. These changes seem unrelated to the task requirements.

4. **tsconfig.json**: Configuration for TypeScript has been updated. This is likely done to resolve modules correctly, but it's not related to the task requirements.

5. **utils/db.ts**: The MongoDB URI has been updated, likely pointing to a new database for the deployed service. While this is a step towards deployment, it does not fulfill the task requirement of adding the deployed URL to the `readme.md`.

In conclusion, the code changes provided do not cover the task requirements. The deployed app URL needs to be added to the `readme.md` file as per the task requirements.

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

### Task goal
The goal of the task was to deploy the service and add the deployed app URL to the `readme.md` file.

### Review

#### readme.md
The deployed app URL has been added to the `readme.md` file as per the task requirements. This change is correct and meets the task requirements.

```diff
@@ -20,4 +20,8 @@ Express.js + Typescript, MongoDB, Mongoose
 
 ### 📖 How to start this service
 
-1. Run ``yarn server``
\ No newline at end of file
+1. Run ``yarn server``
+
+### Deployed app URL on render.com
+
+https://fischkapp-api-agnbart.onrender.com
\ No newline at end of file
```

#### Other changes
There are several other changes in files `index.ts`, `package.json`, `tests/card-delete.test.ts`, `tests/card-get.test.ts`, `tests/card-post.test.ts`, `tests/card-put.test.ts`, `tsconfig.json`, `utils/db.ts`. However, these changes are not related to the task requirements and therefore are not subject to this review. 

### Conclusion
The task requirements have been met. The deployed app URL has been added to the `readme.md` file.
